cmd/submit.go

```diff
@@ -151,6 +151,15 @@ func runSubmit(cfg config.Config, flags *pflag.FlagSet, args []string) error {
 		if err != nil {
 			return err
 		}
+		const MaxFileSize int64 = 65535
```

Comment:
In Go variable names starting with a capital letter usually denote exported variables (shared across the package). Given the scope in which this constant is defined it can not be exported. 

In an attempt to keep things inline with the Code review section for Go https://github.com/golang/go/wiki/CodeReviewComments#mixed-caps this const should be called `maxFileSize`.